setup.py

```diff
@@ -26,9 +26,8 @@
     "diagnostics": ["bokeh >= 1.0.0"],
     "delayed": ["cloudpickle >= 0.2.2", "toolz >= 0.8.2"],
 }
-extras_require["complete"] = sorted(
-    {"PyYaml"} | {v for req in extras_require.values() for v in req}
-)
```

Comment:
Hi! Was there a reason to remove the "complete" extra? What would be the preferred way to obtain a similar behavior now?

Reply:
Ah sorry, that was a mistake. See #6257.

Reply:
Note that you may not need `complete`, and may be better served by composing a few smaller extras together. For example: `dask[array,dataframe,distributed]`.

Reply:
Worth looking into. I'll do a few local tests. Thanks for the speedy help!